SUI/SUI/ClimateViewModel.swift

```diff
@@ -0,0 +1,52 @@
+// ClimateViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана климата
+final class ClimateViewModel: ObservableObject {
+    // MARK: - Public properties
+
+    @Published var isOnClimate = false
```

Comment:
числовые размещай с числовыми,  булевые с булевыми, будет проще читаться
```suggestion
   @Published var minCelsus: CGFloat = 15
    @Published var maxCelsus: CGFloat = 30
    @Published var currentCelsus = 15
    @Published var isAlertShown = false
    @Published var isOnClimate = false
```